I added more scene management

ColtonKor opened the pull request:
I made it so the game is a full loop now. when we reach the end we actually get sent to the credits scene and get sent back

ColtonKor committed:
I added more scene management

I made it so the game is a full loop now. when we reach the end we actually get sent to the credits scene and get sent back

ColtonKor committed:
I made it so the pause menu doesn't start on

mbelles21 reviewed (approved):
it's all coming together

ColtonKor committed:
I fixed an error with the end scene

When we went there from the main game it would turn the cursor off and we couldn't press the buttons even if we press escape. I fixed that now

CSUMBNathan reviewed (approved):
Gucci
